Convert doc reference from "rake" to "rails"

SeanPrashad opened the pull request:
#### :tophat: What? Why?

This PR updates documentation to reference `rails` instead of `rake` commands for tasks inside the generated app.

#### :pushpin: Related Issues
- Fixes #2386 

#### :clipboard: Subtasks
- N/A

### :camera: Screenshots (optional)
- N/A

#### :ghost: GIF
- N/A

deivid-rodriguez commented:
Hi @SeanPrashad, thanks so much for helping out with this!

We probably failed to explain the issue correctly. We need to update the documentation for tasks _inside_ generated applications. The tasks run from the `decidim` root folder still need to be run via `rake`, since the `decidim` repo itself is not a Rails application and thus the `rails` command is not available.

That all means the only change we actually want to keep would be the one in `decidim-dev/README.md`, and that one should fix this issue. Could you update the PR? :smiley:

xabier commented:
@seanprashad you are more than welcome to move to a more challenging PR next :smile:

SeanPrashad commented:
@deivid-rodriguez PR has been updated! I reverted back the changes in everything but `decidim-dev/README.md` 😁

deivid-rodriguez commented:
@SeanPrashad Almost there but not quite yet! Changes in `README.md` need to be reverted since `bundle exec rails s` is correct there. Note that we `cd` into the application folder before that so those happen in the application folder (and `bundle exec rake s` does not exist anyways :))

SeanPrashad commented:
@deivid-rodriguez I've updated the PR to fix the other lines that I modified which weren't supposed to be touched 😀 ! Please let me know if there is anything else I can do

deivid-rodriguez commented:
Nothing else, thanks!!
